unreleased.rst

```diff
@@ -15,3 +15,18 @@ HEAD — ongoing
   </tools/ledger-api-test-tool/index>` to allow disabling randomization of party
   and command identifiers. It is useful for testing of ledgers which are
   configured with a predefined static set of parties.
+- [DAML-LF]: Release version 1.6. This versions provides:
+
+  + ``enum`` types.  See `#105 <https://github.com/digital-asset/daml/issues/105>`__.
```

Comment:
We should give a quick explanation how people need to adapt their applications written against the Ledger API (and codegens) regarding enum types.

Reply:
working on it.